src/constants.ts

```diff
@@ -52,7 +52,7 @@ export const networks = [{
     blockExplorerUrls: ['https://polygonscan.com/']
   },
   blockExplorer: 'https://polygonscan.com/',
-  graphUrl: 'https://api.thegraph.com/subgraphs/name/beehive-innovation/rain-protocol-v2-polygon'
+  graphUrl: 'https://api.thegraph.com/subgraphs/name/beehive-innovation/rain-protocol-polygon-e590ce'
```

Comment:
Is there a reason we still have the subgraph urls in the constants? Can't they come directly from the sdk now? @rouzwelt @nagaremayur

Reply:
no actually, it is not being used anywhere, I'll clean it up